providers/onesignal/src/lib/onesignal.provider.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  IPushOptions,
+  IPushProvider,
+} from '@novu/stateless';
+import * as OneSignal from 'onesignal-node';
```

Comment:
Ok, I understand why you went for the naming pattern, though we have a different standard based on their branding.